### PR TITLE
Clean up, start adding tests, switch to Node 8 (and get rid of Babel for now)

### PR DESCRIPTION
**Pull request by brownjava**
Initial attempt to start cleaning up things in classy-lambda-common a bit to get ready to convert it into a generic utilities / foundational package.

### CODE REVIEW

File: src/PayClient.js [hunk 1]
```diff
@@ -0,0 +1,212 @@
+const Promise = require('bluebird');
+const req = require('requestretry');
```
Comment (tomborger) [inline src/PayClient.js]: One problem we've seen with FRS is the client retrying a donation after getting an ETIMEDOUT or something similar on the first try (usually due to the payment processor itself getting bogged down), and ending up being double-charged. Seems like automatically retrying any request someone sends to Pay through this client might open the door to similar errors.

File: src/aws/handler.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+const AWSConfig = require('./AWSConfig');
+const bugsnagFactory = require('../utils/bugsnagFactory');
+
+const handlerGenerator = (handler, appName) => {
+  let bugsnag;
+  let config;
+
+  return async (event, context) => {
+    if (!awsConfig) {
```
Comment (tomborger) [inline src/aws/handler.js]: Should `awsConfig` be `config`?

File: src/PayClient.js (hunk 1 above)
Comment (brownjava) [reply src/PayClient.js]: I talked to Jon about this.  There was a reason he used requestretry ages ago but we don't think it applies any more.  I'll switch this to just request.

File: src/aws/handler.js (hunk 2 above)
Comment (brownjava) [reply src/aws/handler.js]: Good catch; will fix momentarily.

**Comment by brownjava** [comment]
I'm abandoning this.  I've placed the changes in the new classy-pay-core project instead, and deprecating classy-lambda-common.

**Pull request closed without merging**